packages/react/src/theme/shared-css.ts

```diff
@@ -19,12 +19,11 @@ export const cssFocusVisible = css({
   variants: {
     isFocusVisible: {
       true: {
-        outline: "transparent solid 2px",
```

Comment:
`outline: "none"` might be better, I don't see outline being unset anywhere in the button component, for example

Reply:
Thanks for the suggestion, I put `outline: "none"` outside the variants